modules/compliance-apply-remediation-for-customized-mcp.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: PROCEDURE
+[id="complianc-operator-apply-remediation-for-customized-mcp"]
+= Applying remediation when using customized Machine Config Pools
+
+When you create a custom `MachineConfigPool`, add a label to the `MachineConfigPool`.
+
+.procedure
+
+. Add a label to your MCP.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: <machine_config_pool_name>
+  labels:
+    pools.operator.machineconfiguration.openshift.io/<machine_config_pool_name>: '' <1>
+----
+<1> `labels` field defines label name to add for MCP.
```

Comment:
I think you need a word before `labels` like "The `labels` field..."

Reply:
addressed